---
# Display name
title: Liang Chen

# Is this the primary user of the site?
superuser: true

# Role/position
role: Professor of Biological Science

# Organizations/Affiliations
organizations:
- name: University of Southern California
  url: "https://www.qcb-dornsife.usc.edu/"

# Short bio (displayed in user profile at end of posts)
bio: Liang Chen is a full professor of quantatitive and computational biology (QCB) at University of Southern California. 
interests:
- High-throughput sequencing data analysis
- mRNA alternative splicing
- Genome-wide association studies

education:
  courses:
  - course: PhD
    institution: Yale University
    year: 2006
  - course: BSc
    institution: Tsinghua University 2001
    year: 2001

# Social/Academic Networking
# For available icons, see: https://sourcethemes.com/academic/docs/page-builder/#icons
#   For an email link, use "fas" icon pack, "envelope" icon, and a link in the
#   form "mailto:your-email@example.com" or "#contact" for contact widget.
social:
- icon: envelope
  icon_pack: fas
  link: 'mailto:liang.chen@usc.edu'
# - icon: twitter
#   icon_pack: fab
#   link: https://twitter.com/GeorgeCushen
- icon: google-scholar
  icon_pack: ai
  link: https://scholar.google.com/citations?user=ztru0eUAAAAJ&hl=en
# - icon: github
#   icon_pack: fab
#   link: https://github.com/gcushen
# Link to a PDF of your resume/CV from the About widget.
# To enable, copy your resume/CV to `static/files/cv.pdf` and uncomment the lines below.
# - icon: cv
#   icon_pack: ai
#   link: files/cv.pdf

# Enter email to display Gravatar (if Gravatar enabled in Config)
email: "liang.chen@usc.edu"

# Highlight the author in author lists? (true/false)
highlight_name: true

# Organizational groups that you belong to (for People widget)
#   Set this to `[]` or comment out if you are not using People widget.
user_groups:
- Principal Investigators
---

My long-term research goal is to develop statistical and computational methods to discover the underlying principles of gene expression regulation in eukaryotes, and to explore how variations or defects in gene regulation cause phenotypic variation or diseases. How a cell controls its gene expression is one of the most fundamental and interesting questions in various biological processes, from intrinsic developmental programs to responses to extrinsic stimuli. Meanwhile, a large majority of genetic variants reported in the genome-wide association studies of common human diseases lies in introns or intergenic regions, suggesting their roles in gene expression regulation instead of protein coding. Thus, it is not surprising that gene expression is tightly regulated and coordinated at multiple levels. At the transcriptional level, the interactions between transcription factors and DNA binding motifs play pivotal roles in transcription initiation. So do epigenetic effects, including histone modifications and DNA modifications. At the post-transcriptional level, mRNA processing, mRNA nucleus-cytosol transport, mRNA degradation, and translational control all increase the complexity of gene expression regulation. For example, alternative splicing generates multiple transcript isoforms from the same gene locus through different combinations of splice sites. RNA editing can change the genetic information encoded in mRNA by altering the nucleotide composition. The nucleus-cytosol transport determines the fraction of mRNA that can be translated to protein products. Accurate quantification and comparison of transcriptomes are needed for each of these steps. My current research focuses on transcriptome analysis (e.g., RNA-seq data analysis) and post-transcriptional regulation analysis (e.g., alternative splicing analysis). In addition, I am interested in genome-wide association studies to identify genetic variants underlying diseases or other quantiative traits.
